content/blogs/2023-07-10-release-0-10-blueprints-worker-groups-scripts.md

```diff
@@ -0,0 +1,299 @@
+
+```yaml
+id: magicDAG
+namespace: dev
+tasks:
+  - id: dag
+    type: io.kestra.core.tasks.flows.Dag
+    tasks:
+      - task:
+          id: customers
+          type: io.kestra.plugin.fs.http.Download
+          uri: https://raw.githubusercontent.com/dbt-labs/jaffle_shop/main/seeds/raw_customers.csv
+      - task:
+          id: orders
+          type: io.kestra.plugin.fs.http.Download
+          uri: https://raw.githubusercontent.com/dbt-labs/jaffle_shop/main/seeds/raw_orders.csv
+      - task:
+          id: payments
+          type: io.kestra.plugin.fs.http.Download
+          uri: https://raw.githubusercontent.com/dbt-labs/jaffle_shop/main/seeds/raw_payments.csv
+      - task:
+          id: transform
+          type: io.kestra.plugin.dbt.cli.Build
+        dependsOn:
+          - customers
+          - orders
+          - payments
+```
+
+When comparing the workflow code and the topology view, you can see how the DAG task automatically designated the first three tasks to run in parallel because they have no upstream dependencies. The transformation task must wait for the successful completion of the dependent tasks, defined by referencing the upstream task IDs inside the `dependsOn` property.
+
+
+## Basic Authentication
+
+This release brings an important addition to Kestra's open-source version: a basic authentication feature. This enhancement strengthens your instance's security, making the open-source Kestra server more suitable for production applications.
+
+Here is how to set a basic authentication user in your Kestra configuration:
+
+```yaml
+kestra:
+  server:
+    basic-auth:
+      enabled: false
+      username: admin
+      password: *******
+```    
+
+## Secret Function
+
+Furthermore, we're introducing a `secret()` function to securely retrieve sensitive values in your workflows using special environment variables — key-value pairs, where the key starts with the `SECRET_` prefix and the value is base64-encoded. The `secret()` function will look up only environment variables with that prefix, and base64-decode the values at runtime.
+
+The example below uses the `secret()` function to retrieve the value of a Slack webhook key:
+
+```yaml
+id: getSecret
+namespace: dev
+tasks:
+  - id: secret
+    type: io.kestra.core.tasks.debugs.Return
+    format: "{{ secret('SLACK_WEBHOOK_KEY') }}"
+```    
+
+Even though the environment variable name is prefixed with `SECRET_`, you only need to reference the key without a prefix in your flow. For more information, check the detailed [Managing Secrets](../docs/05.developer-guide/10.secrets.md) guide.
+
+
+## Improved Polling Triggers
+
+Kestra 0.10.0 significantly improves polling triggers to enhance performance, strengthen security, and simplify maintenance.
+Until now, the trigger evaluation process was handled directly by the Scheduler. The Worker and the Scheduler needed access to external systems, leading to high memory and CPU consumption.
```

Comment:
Ludo gave the following feedback on the release note : 

Polling Triggers
leading to increased memory and CPU consumption.: it's not the right reason, the right reason is
that you have 2 services to secure since they could accept the data
the scaling is more simple since you only have the scale the worker

Maybe changing the last part "leading to high memory and CPU comsuption" 🤔